SwingTimerAPI.lua

```diff
@@ -275,8 +298,13 @@ function lib:SwingEnd(hand)
 	self:Fire("SWING_TIMER_STOP", hand)
 	if (self.casting or self.channeling) and self.isAttacking and hand ~= "ranged" then
 		local now = GetTime()
-		self:SwingStart(hand, now, true)
-		self:Fire("SWING_TIMER_CLIPPED", hand)
+		if isRetails and hand == "mainhand" then		
+			self:SwingStart(hand, now, true)
+			self:Fire("SWING_TIMER_CLIPPED", hand)
+		elseif not isRetails then
```

Comment:
avoid negative tests for game version, better be explicit on which version code block should run